.github/workflows/spike-openocd-tests.yml

```diff
@@ -125,7 +126,30 @@ jobs:
               --gcc /opt/riscv/toolchain/bin/riscv-none-elf-gcc \
               --gdb /opt/riscv/toolchain/bin/riscv-none-elf-gdb \
               --sim_cmd /opt/riscv/spike/bin/spike \
-              --server_cmd $GITHUB_WORKSPACE/src/openocd
+              --remotelogfile-enable \
+              --server_cmd "$GITHUB_WORKSPACE/src/openocd -d3"
+
+      - name: Run Spike64-2-hwthread Tests
```

Comment:
Now that the number of of tests has increased, it may be worth to split them to multiple jobs that would then run in parallel, leading to overall shorter test time.

Reply:
yes, this is definitely a way to go. Will investigate how to do this.

Reply:
It looks to me that the current job can be split to multiple parallel jobs quite easily using [strategy-matrix feature](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow) of Github Actions.

Here is a concrete example of this feature from a different repository where I have used it: https://github.com/HonzaMat/PyOpenocdClient/blob/main/.github/workflows/integration_tests.yml#L7